osmoutils/accum/accum.go

```diff
@@ -77,7 +77,7 @@ func (accum *AccumulatorObject) UpdateAccumulator(amt sdk.DecCoins) {
 // The position is initialized with empty unclaimed rewards
 // If there is an existing position for the given address, it is overwritten.
 func (accum AccumulatorObject) NewPosition(name string, numShareUnits sdk.Dec, options *Options) error {
-	return accum.newPosition(name, numShareUnits, accum.value, options)
+	return accum.NewPositionCustomAcc(name, numShareUnits, accum.value, options)
```

Comment:
Note: changes related to `NewPosition` are a drive-by change where I reduced the API count by removing `newPosition`.

Basically, there was no need for the unexported `newPosition` since `NewPositionCustomAcc` takes exactly the same parameters and can be used in place of `newPosition`.